tinytuya/scanner.py

```diff
@@ -1615,6 +1616,73 @@ def _display_status( item, dps, term ):
             print("    %s[%-25.25s] %s%-18s - DPS: %r" %
                   (term.subbold, name, term.dim, ip, dps))
 
+def _snapshot_load_item( itm ):
+    # normalize all the fields
+    item['id'] = item['gwId'] = _get_gwid( itm )
```

Comment:
BUG: "item" should be "itm" - I hope that wasn't my bad influence. I'll commit a fix.